utils/getLiquidationPrice.ts

```diff
@@ -0,0 +1,20 @@
+export function getLiquidationPrice(
```

Comment:
cool to make this a util. There is a BUNCH of other logic we could pull out into common utils like this. I've created an issue to track this: #64

Reply:
Definitely. For example I repeat the calculation `collRequirementFromWei` a lot because the `collReq` in EMPState container is currently stored in Wei form.